src/components/HorizontalCard.tsx

```diff
@@ -33,20 +13,25 @@ export interface IProps {
 
 const HorizontalCard: React.FC<IProps> = ({
   emoji,
-  title,
-  description,
   children,
   className,
   emojiSize,
 }) => (
-  <StyledCard className={className}>
+  <Box
+    display="flex"
+    flexGrow="0"
+    flexShrink="1"
+    flexBasis="75%"
+    borderRadius="base"
+    className={className}
+  >
     <Emoji size={emojiSize} text={emoji} />
-    <Content>
-      <Title>{title}</Title>
-      <Description>{description}</Description>
+    <Box display="flex" ml="8">
+      <Text fontSize="lg" />
+      <Box as="p" p="0.8" mt="-4" mb="2" />
```

Comment:
Lets use `Text` here.

Reply:
Use `Text` instead of `Box`?

Reply:
Yes, here you are using a Box to print a `p`. We can just use `Text` (from Chakra https://chakra-ui.com/docs/components/text/usage) which already print a `p` tag for you.